jni/Android.mk

```diff
@@ -3,6 +3,15 @@ LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 
 ENABLE_TFLITE_BACKBONE := 1
+ENABLE_TFLITE_INTERPRETER := 1
+
+NEED_TF_LITE := 0
+
+ifeq ($(ENABLE_TFLITE_BACKBONE), 1)
+NEED_TF_LITE := 1
```

Comment:
how about..
``` 
ifeq ($(ENABLE_TFLITE_BACKBONE) $(ENABLE_TFLITE_INTERPRETER), 1)
NEED_TF_LITE :=1
endif
```

Reply:
Tested on my local machine, `$(ENABLE_TFLITE_BACKBONE) $(ENABLE_TFLITE_INTERPRETER)` evaluates to `1 1` so `ifeq` does not work :(

Reply:
is it? they say it is working.

Reply:
I think I need to understand the code. let me improve this later, thank you for the suggestion!